src/Quartz/JobBuilder.cs

```diff
@@ -269,6 +271,17 @@ public JobBuilder WithDescription(string? description)
             return this;
         }
 
+        /// <summary>
+        /// Set the JobType Serialized name
+        /// </summary>
+        /// <param name="jobTypeSerialized">the serialized job Type</param>
+        /// /// <returns>the updated JobBuilder</returns>
+        public JobBuilder WithJobTypeSerialized(string? jobTypeSerialized)
```

Comment:
this should probably be `OfType(string typeName)` to use same naming and overloads as passing Type, JobType and Type should be interchangeable concepts